clang/lib/CIR/CodeGen/CIRGenBuiltin.cpp

```diff
@@ -128,19 +128,31 @@ static mlir::Value emitBinaryMaybeConstrainedFPBuiltin(CIRGenFunction &CGF,
 }
 
 template <typename Op>
-static RValue
-emitBuiltinBitOp(CIRGenFunction &CGF, const CallExpr *E,
-                 std::optional<CIRGenFunction::BuiltinCheckKind> CK) {
+static RValue emitBuiltinBitOp(
+    CIRGenFunction &CGF, const CallExpr *E,
+    std::optional<CIRGenFunction::BuiltinCheckKind> CK = std::nullopt,
+    std::optional<bool> is_zero_poison = std::nullopt,
```

Comment:
We don't have to make this parameter a `std::optional`; `bool` should be sufficient.